agent/scripts/newrelic.ini.template

```diff
@@ -1169,17 +1169,17 @@ newrelic.daemon.logfile = "/var/log/newrelic/newrelic-daemon.log"
 ; Setting: newrelic.framework.wordpress.hooks.options
 ; Type   : string (all_callbacks, plugin_callbacks, threshold)
 ; Scope  : per-directory
-; Default: all_callbacks
+; Default: plugin_callbacks
 ; Info   : Sets the options how WordPress hooks are instrumented.
 ;
 ;          New Relic agent can provide different levels of insights into WordPress hooks.
-;          By default, all hook callbacks functions are instrumented ("all_callbacks").
-;          To reduce agent's overhead it is possible to limit the instrumentation to only
-;          plugin/theme callbacks ("plugin_callbacks"). Third option is to monitor hooks
-;          without instrumenting callbacks ("threshold"). This option does not give insights
-;          about plugins/themes.
+;          By default, only plugin/theme callbacks are instrumented ("plugin_callbacks").
+;          At the cost of increased agent's overhead it is possible to extend the
+;          instrumentation to all hook callbacks functions ("all_callbacks")nly. Third option
```

Comment:
Something seems off on this line as it ends to ")nly."

Reply:
```suggestion
;          instrumentation to all hook callbacks functions ("all_callbacks"). Third option
```